_posts/2023-07-25-introducing-extensions-for-opensearch.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "Introducing extensions for OpenSearch"
+authors:
+  - vemsarat
+  - widdisd
+  - kazabdu
+  - dblock
+  - dagney
+  - minalsha
+  - kolchfa
+date: 2023-07-25
+categories:
+  - technical-posts
+Meta_keywords: OpenSearch extensions, OpenSearch plugins, OpenSearch extensibility
+Meta_description: Discover how the experimental extensions feature in OpenSearch 2.9 helps extend OpenSearch and independently scale workloads without impacting cluster availability.
```

Comment:
See above.

Reply:
Sorry about that, Nate. @kolchfa-aws - are you able to make that quick change